### ISSUE TITLE
Diet Favorite Button

### ISSUE THREAD

**Issue by lenzloesche** [lenzloesche/capstone-project#43]
## Value Proposition

**As a user** 
**I want to favorite a food** 
**so that I will rembmer it**
​

## Description

... (Wireframe, scribble, design or text)
​

## Acceptance Criteria

- [ ] Describe every scenario in detail
- [ ] e.g. all inputs are required, show warning if empty.
- [ ] ...
      ​

## Tasks

- [ ] list the tasks that need to be performend within the dev team
- [ ] e.g. Components, Storybook, testing
- [ ] ...
      ​

## Complexity

... (T-Shirt Sizes XS to XL)

**Comment by reyk-zepper** [comment]
looks like many things going around here...i understand what u want to have done...for myself it is better with a small sribble, even it shows a small cutout. if you give me this userStory at a work order, i'm not shure if i will produce the correct product you order.
but all in all...it is a personal perspective of mine and it is clear out of this paper what u want to have done...thx

**Comment by lenzloesche** [comment]
> looks like many things going around here...i understand what u want to have done...for myself it is better with a small sribble, even it shows a small cutout. if you give me this userStory at a work order, i'm not shure if i will produce the correct product you order. but all in all...it is a personal perspective of mine and it is clear out of this paper what u want to have done...thx

yes, my habit is to write too big userStories. good observation!
